openmetadata-ui/src/main/resources/ui/src/components/Users/ChangePasswordForm.tsx

```diff
@@ -0,0 +1,97 @@
+import { Form, Input, Modal } from 'antd';
+import { isEmpty } from 'lodash';
+import React, { useMemo } from 'react';
+import { ChangePasswordRequest } from '../../generated/auth/changePasswordRequest';
+
+type ChangePasswordForm = {
+  visible: boolean;
+  onCancel: () => void;
+  onSave: (data: ChangePasswordRequest) => void;
+};
+
+const ChangePasswordForm: React.FC<ChangePasswordForm> = ({
+  visible,
+  onCancel,
+  onSave,
+}) => {
+  const [form] = Form.useForm();
+  const newPassword = Form.useWatch('newPassword', form);
+
+  const validationMessages = useMemo(
+    () => ({
+      required: '${label} is required',
+    }),
+    []
+  );
```

Comment:
You can use it from: https://github.com/open-metadata/OpenMetadata/blob/92aea1c1798679a1318950d2233840d74296dfea/openmetadata-ui/src/main/resources/ui/src/constants/auth.constants.ts#L16